src/components/about-us/section-03/content.js

```diff
@@ -163,71 +177,156 @@ const MoreInfo = styled.div`
+    transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(90deg)'};
+    margin-top: ${props => props.unfold ? '0' : 'calc(-204px * 2)'};
+    will-change: transform, margin-top;
+    &:before {
+      content: '';
+      background-image: linear-gradient(to bottom, ${colors.gray.gray64}, ${colors.gray.gray96} 80%);
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: ${props => props.unfold ? '0' : '100%'};
+      z-index: calc(${defaultZIndex} - 1);
+      transition: height .25s;      
+    }
+  }
+`
+
+const MockRecord = styled.div`
+  width: 100%;
+  position: relative;
+  min-height: 204px;
+  height: ${props => props.unfold ? 'auto' : '204px'};
+  transition: all ${transitionDuration} linear;
+  padding: 20px 21px 19px 15px;
+  display: ${props => props.isOdd ? 'block' : 'none'};
+  visibility: ${props => props.isOdd ? 'visible' : 'none'};
+  margin-top: ${props => props.unfold ? '-204px' : 'calc(-204px * 2)'};
+  transform-origin: 50% 100% 0;
+  transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(90deg)'};
+`
+
+const AwardName = styled.div`
+  width: 100%;
+  border: solid 1px ${mobileBorderColor};
+  min-height: ${props => props.unfold ? '53px' : '76px'};
+  margin-top: ${props => props.unfold ? '30px' : 0};
+  background: ${props => props.unfold ? `${colors.black}` : `${colors.white}`};
+  color: ${props => props.unfold ? `${colors.white}` : `${colors.black}`};
+  p{
+    line-height: 76px;
+    text-align: center;
+    font-size: 20px;
+    font-weight: bold;
+    letter-spacing: 1.8px;
+  }
+`
+
+const SeperatedLine = styled.div`
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  height: 50%;
+  border-bottom: solid 0.5px ${colors.black};
   ${screen.tablet`
-    padding-bottom: 10px;
+    width: 65%;
   `}
   ${screen.mobile`
-    font-size: 14px;
-    padding-bottom: 21px;
-  `}
-  ${screen.desktopAbove`
-    display: none;
+    width: 80%;
   `}
 `
 
-const PagesWrapper = styled.div`
+const YearTag = styled.div`
   position: relative;
-  width: 100%;
-  padding: 40px 46px;
-  ${screen.desktopAbove`
-    height: 100%;
-    overflow-y: hidden;
-  `}
+  background: ${colors.white};
+  display: ${props => props.unfold ? 'block' : 'none'};
+  height: 65px;
+  p{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateX(-50%) translateY(-50%);
+    display: inline-block;
+    background: ${colors.white};
+    color: ${colors.black};
+    font-family: ${font.family.english.roboto}, ${font.family.sansSerifFallback};    
+    font-size: 18px;
+    font-weight: ${font.weight.bold};
+    z-index: calc(${defaultZIndex} + 1);
+    padding: 0 5px;
+  }
   ${screen.tablet`
-    padding: 0 116px 20px 31px;
-  `}
-  ${screen.mobile`
-    padding: 0 7px 29px 0; 
+    p{
+      padding: 0 10px;
+    }
   `}
 `
 
 export default class Content extends PureComponent {
```

Comment:
Could we separate mobile and desktop version into different React component? 
It's kind of complex for me to understand the structure of HTML.

We need to keep it simple as possible.